src/components/shared/EthToUsd.tsx

```diff
@@ -0,0 +1,23 @@
+import { BigNumber } from 'ethers'
+import { useCurrencyConverter } from 'hooks/v1/CurrencyConverter'
+import { formatWad } from 'utils/formatNumber'
+
+import Loading from './Loading'
+
+// Takes an ETH amount and returns equiv in USD
+export default function EthToUsd({
+  ethAmount,
+}: {
+  ethAmount: BigNumber | string
+}) {
+  const converter = useCurrencyConverter()
+  const usdAmount = converter.wadToCurrency(ethAmount, 1, 0)
+  const usdAmountFormatted = formatWad(usdAmount, {
+    precision: 2,
+    padEnd: true,
+  })
+  if (usdAmount?.gt(0)) {
+    return <span>{usdAmountFormatted} USD</span>
```

Comment:
**suggestion**: thoughts on`US$x` instead?

> It is represented by the symbol $ or US$ to differentiate it from other dollar-based currencies.

https://www.investopedia.com/terms/forex/u/usd-united-states-dollar.asp

Reply:
could go either way on this one. US$ X does seem nicer since it looks more like a currency value, though X USD seems more uniform with X ETH.

maybe we could replace cases of $x with US$x, so depending on where it's being used the convention would be:
- Ξx / US$x
- x USD / x ETH